### PR TITLE
Fix styleck errors

### PR DESCRIPTION
**Pull request by adamjsawicki**

### CODE REVIEW

**Comment by adamjsawicki** [comment]
This should make all travis tests pass allowing ipfs/py-datastore#13 to be accepted.

File: datastore/adapter/tiered.py
Comment (ntninja) [inline datastore/adapter/tiered.py]: Why drop the category label here? (I try to be as specific as possible with mypy typing exceptions, so that potential other problems in the same line aren't missed.)

File: datastore/core/util/stream.py
Comment (ntninja) [inline datastore/core/util/stream.py]: Same here

File: datastore/core/util/stream.py
Comment (ntninja) [inline datastore/core/util/stream.py]: Split this declaration across several lines

File: datastore/core/util/stream.py
Comment (ntninja) [inline datastore/core/util/stream.py]: [category label]

File: datastore/core/util/stream.py
Comment (ntninja) [inline datastore/core/util/stream.py]: [category label]

File: datastore/core/util/stream.py
Comment (ntninja) [inline datastore/core/util/stream.py]: [multiline declaration]

File: datastore/core/util/stream.py
Comment (ntninja) [inline datastore/core/util/stream.py]: [category label]

File: tox.ini
Comment (ntninja) [inline tox.ini]: There is no `setup.py` file or `other_project` directory? (Also what is warning category W9?)

File: tox.ini
Comment (ntninja) [inline tox.ini:60]: Please add a comment that we allow F401, which is “module imported but not used”, because these modules only exist to expose the package's public API.

**Review by ntninja — CHANGES_REQUESTED** [review]
Thank you! I left some comments or parts that I don't understand or like to see changed, but most stuff perfectly fine.

File: datastore/adapter/tiered.py
Comment (adamjsawicki) [reply datastore/adapter/tiered.py]: flake8 was giving 
```
datastore/adapter/tiered.py:52: F821 undefined name 'ignore'
datastore/adapter/tiered.py:52: F821 undefined name 'assignment'
```

I can do a `# noqa: F821` on all these instances if you think that's preferred.

EDIT: Seems this is an error with `pyflakes` that will be fixed in next major release. See [here](https://github.com/PyCQA/pyflakes/issues/373#issuecomment-535887835) for discussion, but seems using `noqa` is solution for now.

**Review by ntninja — APPROVED** [review]
Thank you! I'll merge this once the base branch has been merged in `ipfs/py-datastore`.

Don't know what to do about that `flake8` bug, but your solution is probably the best available for the time being.

**Pull request merged**
